frontend/ui/src/app/licenses/edit-license.component.html

```diff
@@ -94,19 +94,53 @@
+                          [value]="item"
+                          class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
+                        <label
+                          for="version-checkbox-{{ i }}"
+                          class="w-full py-3 ms-2 text-sm font-medium text-gray-900 dark:text-gray-300">
+                          {{ itemLabel }}
+                        </label>
+                      </div>
+                    </li>
+                  }
+                }
+                @let archived = includedArchivedVersions();
+                @if (archived.length > 0) {
                   <li class="w-full border-t border-gray-200 dark:border-gray-600">
-                    <div class="flex items-center ps-3">
-                      <input
-                        id="version-checkbox-{{ i }}"
-                        type="checkbox"
-                        [formControlName]="i"
-                        [value]="item"
-                        class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
-                      <label
-                        for="version-checkbox-{{ i }}"
-                        class="w-full py-3 ms-2 text-sm font-medium text-gray-900 dark:text-gray-300">
-                        {{ itemLabel }}
-                      </label>
+                    <div class="flex items-center ps-3 py-3 text-sm font-medium text-gray-900 dark:text-gray-300">
+                      <details class="open:border-gray-200 dark:open:border-gray-700 rounded-lg text-sm font-medium">
+                        <summary class="select-none">Including {{ archived.length }} Archived Versions</summary>
+                        <ul
+                          class="w-full mt-1 text-sm font-medium text-gray-900 bg-white border-gray-200 rounded-lg dark:bg-gray-700 dark:border-gray-600 dark:text-white">
+                          @for (av of archived; track av.id; let i = $index) {
+                            <li class="w-full border-gray-200 dark:border-gray-600">
+                              <div class="flex items-center">
+                                <input
+                                  type="checkbox"
+                                  id="archived-checkbox-{{ i }}"
+                                  checked
+                                  disabled
+                                  class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
+                                <label
```

Comment:
To further indicate that this checkbox is disabled, I would make the text a little more muted, maybe copypasta from here: https://flowbite.com/docs/forms/checkbox/#disabled-state